tools/compiler.js

```diff
@@ -10,6 +10,7 @@ var meteorNpm = require('./meteor-npm.js');
 var watch = require('./watch.js');
```

Comment:
We should do an extra pass over this file to make sure we are converting all of the ins and outs properly